### PR TITLE
Refactored tunnel experiments

### PR DESCRIPTION
**Pull request by devos50**
This PR refactors tunnel experiments and fixes some issues with running Tribler within Gumby. Depends on https://github.com/Tribler/tribler/pull/3391 (should be merged first).

### CODE REVIEW

**Comment by devos50** [comment]
retest this please

**Comment by devos50** [comment]
@qstokkink could you review this PR? The pylint seems to have some issues but I have no idea how to solve that. The Tribler integration test succeeds so I don't think it's a major issue.

**Comment by qstokkink** [comment]
@devos50 Pylint doesn't have the source directories configured. That's why you get the import errors. `Tribler` isn't part of the `Gumby` package at compile time.

File: experiments/tunnels/parse_tunnel_statistics.py [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+    def __init__(self, node_directory):
+        self.node_directory = node_directory
+        self.circuits_info = []
+        self.relays_info = []
+
+    def yield_files(self, file_to_check='market_stats.log'):
+        pattern = re.compile('[0-9]+')
+
+        # TODO: this only contains support for a localhost structure
```
Comment (qstokkink) [inline experiments/tunnels/parse_tunnel_statistics.py]: Ehh?

**Review by qstokkink — APPROVED** [review]

**Pull request merged**